backend/service/db/postgres/setup.go

```diff
@@ -51,6 +51,10 @@ func New(cfg *any.Any, logger *zap.Logger, scope tally.Scope) (service.Service,
 		return nil, err
 	}
 
+	if pgcfg.MaxIdleConnections > 2 {
+		sqlDB.SetMaxIdleConns(int(pgcfg.MaxIdleConnections))
```

Comment:
I want to write some type of test for this but there are not Getter methods to assert this value. I tried to extract the value via the stats that are exposed without luck.

Reply:
yeah, we'd have to replace sql.DB with an interface so you could inject a mock. maybe we should at some point.